perf/datarepo/datarepo-api.yaml

```diff
@@ -28,6 +28,8 @@ env:
   OIDC_ADDCLIENTIDTOSCOPE: true
   OIDC_EXTRAAUTHPARAMS: prompt=login
   SAM_OPERATIONTIMEOUTSECONDS: 120
+  TPS_ENABLED: true
+  TPS_BASE_PATH: https://tps.dsde-dev.broadinstitute.org
```

Comment:
Just want to make sure: will there be an integration or perf deployment for TPS?

Reply:
I'll double check with the identiteam!